Add guide to configure `process_file_events` on CentOS 7

lucasmrod opened the pull request:
#11890

rachaelshaw commented on docs/Using-Fleet/Process-File-Events.md:
````diff
@@ -0,0 +1,178 @@
+echo "other" > /etc/foobar/other.txt
+```
+
+### 4. Create a test team in Fleet.
````
```suggestion
## 4. Create a test team in Fleet.
```

rachaelshaw commented on docs/Using-Fleet/Process-File-Events.md:
````diff
@@ -0,0 +1,178 @@
+rpm --install fleet-osquery-1.10.0.x86_64.rpm
+```
+
+### 6. Set team agent options
````
```suggestion
## 6. Set team agent options
```

rachaelshaw commented on docs/Using-Fleet/Process-File-Events.md:
```diff
@@ -0,0 +1,178 @@
+
+### 6. Set team agent options
+
+Set the following settings on the team's agent options:
```
```suggestion
Configure the following settings on the team's agent options:
```

rachaelshaw commented on docs/Using-Fleet/Process-File-Events.md:
```diff
@@ -0,0 +1,178 @@
+- `audit_backlog_limit: 60000`: Sets the queue length for audit events awaiting transfer to osquery audit subscriber. We set this to a high value first to make sure the table is working, then it should be modified to a better value suited for production.
+- The following flags were set to `false` to avoid unnecessary load on the host: `audit_allow_process_events: false`, `audit_allow_sockets: false`, `audit_allow_user_events: false`, `audit_allow_selinux_events: false`, `audit_allow_kill_process_events: false`, `audit_allow_apparmor_events: false`, `audit_allow_seccomp_events: false`, `enable_bpf_events: false`.
+
+### 7. Make sure osquery audit subscriber is working
```
```suggestion
## 7. Make sure osquery audit subscriber is working
```

rachaelshaw commented on docs/Using-Fleet/Process-File-Events.md:
```diff
@@ -0,0 +1,178 @@
+
+`enabled` should be `1` and `pid`'s value should be the process ID of osquery.
+
+### 8. Modify the test files
```
```suggestion
## 8. Modify the test files
```

rachaelshaw commented on docs/Using-Fleet/Process-File-Events.md:
```diff
@@ -0,0 +1,178 @@
+> IMPORTANT: Reminder that the files must exist before the osquery process is initialized.
+> Creating or modifying new files won't generate `process_file_events` events.
+
+### 9. Query the process_file_events table
```
```suggestion
## 9. Query the process_file_events table
```

rachaelshaw commented on docs/Using-Fleet/Process-File-Events.md:
```diff
@@ -0,0 +1,178 @@
+
+It should return two events, one with `operation=write` and one with `operation=unlink`.
+
+## Misc notes
```
```suggestion
## Additional notes
```

rachaelshaw commented on docs/Using-Fleet/Process-File-Events.md:
````diff
@@ -0,0 +1,178 @@
+auditdnetlink.cpp:354 The Audit publisher has throttled reading records from Netlink for 0.2 seconds. Some events may have been lost.
+```
+Some events might get lost due to system load or low CPU/memory resources.
+
````
We may need a meta tag here, if the page title doesn't end up matching this file's name (`<meta name="title" value="Title of this page">`)

rachaelshaw reviewed (changes requested):
Suggested a bunch of little formatting changes after previewing how this would compile to the website, as well as a few minor verbiage tweaks, but the content looks good overall ✨

lucasmrod commented:
> Suggested a bunch of little formatting changes after previewing how this would compile to the website, as well as a few minor verbiage tweaks, but the content looks good overall ✨

Thank you for the detailed review!

I believe I've addressed all the comments.

lucasmrod commented:
@rachaelshaw Seems you need some permissions to allow me to merge after your approval?
![Screenshot 2023-06-14 at 18 05 23](https://github.com/fleetdm/fleet/assets/2073526/c1c2ffb0-c080-407c-a54d-7143370e6ebe)

rachaelshaw commented:
> @rachaelshaw Seems you need some permissions to allow me to merge after your approval?

Looks like this needs an approval from fleetdm/go because of the other file change, I think that's what's blocking it:
![Screenshot 2023-06-14 at 4 15 37 PM](https://github.com/fleetdm/fleet/assets/3065949/4ad30e81-18e5-477a-afc6-3abfda7fd86a)
(I see you're in that group though, so maybe if you hit approve first it'll let you merge?)

sharon-fdm reviewed (approved):
LGTM
